translation/en/RedirectedResources/assets/abdata/adv/scenario/c26/00/84/translation.txt

```diff
@@ -1,9 +1,9 @@
 //
 // Dumped for Koikatsu v5.1 (CharaStudio) by KK_TextDump v1.4.6.0
 //
-「……コイカツって、何をやるの？」=“...Koikatsu, what is it anyway?”
+「……コイカツって、何をやるの？」=“...What are love activities, anyway?”
```

Comment:
Even if it wasn't the game title, it uses katakana, so it makes more sense to just use `Koikatsu`. Besides the next few lines explain it.
```suggestion
「……コイカツって、何をやるの？」=“...What is Koikatsu, anyway?”
```